### PR TITLE
Update Turkish learn 2 bucket

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Update Learn 2 bucket for Turkish (`tr`) lang

### CODE REVIEW

File: src/content/translations/tr/history/index.md [hunk 1]
```diff
@@ -24,15 +24,37 @@ Gelecekteki protokol yükseltmelerini mi arıyorsunuz? [Ethereum'da yapılacak y
...
 
 #### Özet {#arrow-glacier-summary}
 
-"Arrow Glacier" (Ok Buzulu) ağ yükseltmesi, [zorluk bombasını](/glossary/#difficulty-bomb) bir kaç ay ertelendi. Bu yükseltmede sunulan tek değişiklik budur ve doğası gereği [Muir Glacier](#muir-glacier) yükseltmesine benzer. [Bizans](#byzantium), [Konstantinopolis](#constantinople) ve [Londra](#london) ağ yükseltmelerinde de benzer değişiklikler yapıldı.
+The Arrow Glacier ağ yükseltmesi [zorluk bombasını](/glossary/#difficulty-bomb) bir kaç ay ertelendi. Bu yükseltmede sunulan tek değişiklik budur ve doğası gereği [Muir Glacier](#muir-glacier) yükseltmesine benzer. [Bizans](#byzantium), [Konstantinopolis](#constantinople) ve
+
+Londra</ağ yükseltmelerinde de benzer değişiklikler yapıldı. >.</p>
```
Comment (corwintines) [inline src/content/translations/tr/history/index.md:57]: This looks to be causing some problems still.

File: src/content/translations/tr/glossary/index.md [hunk 2]
```diff
@@ -122,126 +158,178 @@ Yüksek seviyeli bir programlama dilinde yazılmış kodu (ör. [Solidity](#soli
...
+
+### entropi {#entropy}
+
+Kriptografi bağlamında, öngörülebilirlik eksikliği veya rastgelelik düzeyi. [Özel anahtar](#private-key)lar gibi gizli bilgiler üretirken, algoritmalar çıktının tahmin edilemez olmasını sağlamak için genellikle yüksek entropi kaynağına güvenir.
 
 ### dönem {#epoch}
 
-[İşaret Zinciri](#beacon-chain)-koordineli sistemde 32 [yuvalık](#slot) (6,4 dakika) bir periyot. [Doğrulayıcı](#validator) [komiteleri](#committee), güvenlik nedenleriyle her dönem karıştırılır. Zincirin [kesinleştirilmesi](#finality) için her dönemde bir fırsat vardır.
+[İşaret Zinciri](#beacon-chain)-koordineli sistemde 32 [yuvalık](#slot) (6,4 dakika) bir periyot. [Doğrulayıcı](#validator) [komiteler](#committee), her dönem, güvenlik nedenleriyle karıştırılır. Zincirin [sonlandırılması](#finality) için her çağda bir fırsat vardır. Terim ayrıca [yürütme katmanında](#execution-layer) kullanılır ve <a href= iş kanıtı algoritması [Ethash](#Ethash) tarafından tohum olarak kullanılan veritabanının her yenilenmesi arasındaki aralık anlamına gelir. Dönem, 30000 blok olarak belirtilmiştir.
```
Comment (corwintines) [inline src/content/translations/tr/glossary/index.md:303]: Looks like there are issues with the a tag here

Comment (corwintines) [reply src/content/translations/tr/glossary/index.md:303]: @minimalsm i'm not sure we need this a tag at all.

File: src/content/translations/tr/history/index.md (hunk 1 above)
Comment (corwintines) [reply src/content/translations/tr/history/index.md:57]:
```suggestion
Londra ağ yükseltmelerinde de benzer değişiklikler yapıldı.
```
I think this is what we will need, but would double check

File: src/content/translations/tr/glossary/index.md (hunk 2 above)
Comment (minimalsm) [inline src/content/translations/tr/glossary/index.md:303]: Good eyes

```suggestion
[İşaret Zinciri](#beacon-chain)-koordineli sistemde 32 [yuvalık](#slot) (6,4 dakika) bir periyot. [Doğrulayıcı](#validator) [komiteler](#committee), her dönem, güvenlik nedenleriyle karıştırılır. Zincirin [sonlandırılması](#finality) için her çağda bir fırsat vardır. Terim ayrıca [yürütme katmanında](#execution-layer) kullanılır ve iş kanıtı algoritması [Ethash](#Ethash) tarafından tohum olarak kullanılan veritabanının her yenilenmesi arasındaki aralık anlamına gelir. Dönem, 30000 blok olarak belirtilmiştir.
```

**Review by corwintines — APPROVED** [review]
LGTM

**Pull request merged**